docs/CONSOLE.md

```diff
@@ -50,8 +50,15 @@ act/0x77df6ca4cc96d16edc7858cfc00f70fdc98bb027/0xe96c9851773ec7adcb6a155c7d4acf1
 ```
 
 The above command will select the Agent at `0x77df...b027`, interact with the Voting app at `0xe96c...e7ae` (installed on a different organization), and execute the method `vote(uint256 voteId, bool supports, bool executesIfDecided)`.
- 
-Note that as of now, you cannot use `act` to execute actions requiring ETH. If you need this functionality, you can use `exec` and call `execute()` with the encoded calldata.
+
+The act command can receive the amount of ETH to send using the last parameter of the command, in wei. This parameter is optional. You can also use the exec() command with the necessary encoded calldata.
```

Comment:
> You can also use the `exec()` command with the encoded calldata necessary.

Hmm, not sure I understand this, but if we're mentioning using `exec` to call `act`, I would just leave it out now since there's no good reason to do so.

-----

I saw we've already updated the first sentence a bit, but I'd also recommend:

```suggestion
As a final optional parameter, the `act` command can receive the amount of ETH, in wei, to send with the transaction.
```

Reply:
Ah, this is much clearer!